.changeset/great-drinks-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+'eslint-plugin-amplify-backend-rules': minor
+---
+
+Adding two new custom rules and overrides to control how they are applied
```

Comment:
```suggestion
---
---

```

For [private packages](https://github.com/aws-amplify/amplify-backend/blob/150541422f6bcb13fc5b1312cff01791a07cac5c/packages/eslint-rules/package.json#L3) empty changeset should be good enough .

The proposed change is equivalent to running `npx changeset --empty` .